lib/game.rb

```diff
@@ -0,0 +1,71 @@
+class Game
+	attr_accessor :board, :player_1, :player_2
+
+	WIN_COMBINATIONS = [[0,1,2],
+      [3,4,5],
+      [6,7,8],
+      [0,3,6],
+      [1,4,7],
+      [2,5,8],
+      [0,4,8],
+      [6,4,2]]
+
+	def initialize(player_1 = Human.new("X"), player_2 = Human.new("O"), board = Board.new)
+		@player_1 = player_1
+		@player_2 = player_2
+		@board = board
+	end
+
+	def current_player
+		self.board.turn_count % 2 == 0 ? self.player_1 : self.player_2
+	end
+
+	def winner
+		x_array = self.board.cells.map.with_index {|cell,i| i if cell == "X"}
+		o_array = self.board.cells.map.with_index {|cell,i| i if cell == "O"}
+		WIN_COMBINATIONS.each do |combination|
+			return "X" if combination.all? {|num| x_array.include?(num)}
+			return "O" if combination.all? {|num| o_array.include?(num)}
+		end
+		nil
+	end
+
+	def won?
+		if self.winner == "X" || self.winner == "O"
```

Comment:
Here again, the if/else structure is superfluous, because of the implicit return value of the boolean condition.

```
def won?
  self.winner == "X" || self.winner == "O"
end
```